<launch>
    <node pkg="drive" type="checkdata" name="datachecknode" respawn="true"/>
    <node pkg="rosserial_python" type="serial_node.py" name="serial_node" args="/dev/ttyACM0" respawn="true"/>
    <node pkg="drive" type="drive_pub" name="driver" respawn="true"/>
    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="configuration" respawn="true"/>
    <node pkg="drive" type="open_exe" name="displaydata" respawn="true"/>
</launch>
